common/src/test/java/uk/gov/gchq/palisade/policy/PassThroughRule.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.gchq.palisade.policy;
```

Comment:
Ah, I see this is used in testing code below. If this is only intended to support unit tests, would it be better off being inside the test package?

Reply:
So it is in the common test jar due to it needing to be distributed with all the data services for the docker/multi JVM examples